package.json

```diff
@@ -24,10 +24,8 @@
   "files": [
     "target"
   ],
-  "dependencies": {
-    "@types/node": "^12.0.10"
-  },
   "devDependencies": {
+    "@types/node": "^12.0.10",
```

Comment:
Only used by tests, so safe to move here

Reply:
Precisely. Reference tweet for anyone interested:

https://twitter.com/bterlson/status/1100797304632557573